pkg/ims/service/service.go

```diff
@@ -140,4 +148,34 @@ func (service *SignUpService) EmailVerificationToken (token *models.EmailVerific
 	return nil
 }
 
-// Send Email Verification Token function must be here!
\ No newline at end of file
+// Send Email Verification Token function must be here!
```

Comment:
Delete this comment because the Email Verification Token function is created and is located above